cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -803,14 +782,14 @@ public void testNewKeyGetGeneratedWhenInRenewalPeriodAuto(VertxTestContext conte
                 assertThat(newClusterCaKey, is(notNullValue()));
                 assertThat(newClusterCaKey, is(not(initialClusterCaKeySecret.getData().get(CA_KEY))));
 
-                Map<String, String> clientsCaCertData = c.getAllValues().get(2).getData();
+                Map<String, String> clientsCaCertData = secretData(c, 2);
                 assertThat(clientsCaCertData, aMapWithSize(4));
                 X509Certificate newX509ClientsCaCertStore = getCertificateFromTrustStore(CA_CRT, clientsCaCertData);
                 String oldClientsCaCertKey = clientsCaCertData.keySet()
                         .stream()
                         .filter(alias -> alias.startsWith("ca-"))
                         .findAny()
-                        .get();
```

Comment:
This removes the warning about an `Optional#get` without a `Optional#isPresent` call first.